### ISSUE TITLE
Generate Public Key from the Private Key

### ISSUE THREAD

**Issue by burstpay** [ionux/phactor#6]
Hello

Can phactor be used in any way to generate the public key from the private key?

Good job with this project!

Thank you!

**Comment by ionux** [comment]
Hey @burstpay sorry for the delay responding to you.  My wife and I recently had our first baby so things have been pretty busy at home! :)

I'll take another look but I believe the library is already doing what you ask.

**Comment by yupnano** [comment]
Hi @ionux , congratulations on this happy news!

And I have this same question: How to get pulickey from a given privatekey? 
Maybe the library couold already achieve this, however we can't find how to use it from the sample code in the readme. 
Could you please add some examples concerning this issue?

**Comment by jackneil** [comment]
Same question here! Thanks!

**Comment by ionux** [comment]
Hey guys, thanks for writing to me about this and reminding me I need to add this handy method.  I was just looking through my code and saw there's already a method I had written in the past to obtain the private from the public key so it should be fairly trivial to do the same for the opposite.  Give me a few days for this and I'll turn it around for you and publish a new release with this and the other changes since the last release.

Thanks again for reminding me!

**Comment by mattvb91** [comment]
:+1: for this

@jackneil @yupnano @burstpay have you guys made any progress with this?

**Comment by mattvb91** [comment]
Just checking in again

**Comment by mattvb91** [comment]
@jackneil @yupnano @burstpay  @ionux bumping again if anyone has made any progress?

**Comment by meglio** [comment]
Has this been abandoned?

**Comment by ionux** [comment]
Nope, not at all.On Mar 21, 2023, at 7:12 PM, Anton A. ***@***.***> wrote:﻿
Has this been abandoned?

**Comment by mookey86** [comment]
src

**Comment by meglio** [comment]
@mookey86 what do you mean?
